src/components/primitive/notice-loader/notice-loader.component.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useTheme } from 'styled-components';
+import { Image, GenericModal } from 'components/primitive';
+import { LoaderComponentProps } from './notice-loader.component.props';
+import { Spinner, LoaderText, HelperText, NoticeLoaderContainer } from './notice-loader.component.styles';
+
+export const NoticeLoader: React.FC<LoaderComponentProps> = (props) => {
+  const { label, spinner, helperText } = props;
+  const { images } = useTheme();
+
+  return (
+    <GenericModal show={true} hide noClose>
+      <NoticeLoaderContainer>
+        {spinner ? <Spinner /> : <Image src={images.lockLoader} width={7} />}
```

Comment:
Add an optional icon prop that will replace default image/ spinner

Reply:
Done, added in the latest commit.